ui/widgets/tooltip.js

```diff
@@ -140,6 +140,8 @@ $.widget( "ui.tooltip", {
 		}
 	},
 
+	_setOptionDisabled: $.noop,
```

Comment:
I'm also not sure why `._setOption()` has the current implementation; removing the `return` shouldn't change the functionality at all, so we should be able to just remove that conditional.